components/alert/alert.css

```diff
@@ -0,0 +1,51 @@
+.alert__demo {
+  flex-wrap: wrap;
+  gap: 10px;
+  padding: 2rem;
+}
+.alert {
+  align-content: center;
+  padding: 1rem;
+  border-radius: 0.5rem;
+  border: 0.1rem solid;
```

Comment:
for shorter lengths, you can use px instead of relative units

Reply:
i had used those in the earlier PR got reviewed that i should change everything to relative units so changed them to rem instead

Reply:
ohhh ... you can use px for shorter lengths of 10px or lower than it